sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -2846,7 +2846,7 @@ test_suite(
     opentitan_gdb_fpga_cw310_test(
         name = "rom_e2e_debug_test_otp_" + otp_name,
         timeout = "short",
-        exit_success_pattern = "OK!GDB-OK\r\nBFV:0142500d",
+        exit_success_pattern = "OK!GDB-OK(?s:.*)BFV:0142500d",
```

Comment:
I slightly prefer the exactness of the original pattern. Or was it subtly broken because I didn't double-backslash?

(Took me a bit to grok this regex, but I got there by reading the Rust [regex](https://docs.rs/regex/latest/regex/#syntax) docs.)

Reply:
Good point. Let me explain: the output of the ROM with boot failure is:
```
ROM: xxxxxxxx <- Printed towards the end of rom_init()
BFV: xxxxxxxx
LCV: xxxxxxxx
```

Since we now run these gdb commands after `uart_init()` instead of `rom_init()` we should expect `ROM: xxxxxxxx` after `OK!GDB-OK` instead of `BFV:0142500d`. The `ROM: xxxxxxxx` message, however, is only printed on FPGAs (see the definitions of `device_fpga_version_print()` for different platforms). Hence the `.*` between the `OK!GDB-OK` and `BFV:0142500d`.

Reply:
Gotcha. SGTM.